### PR TITLE
Add Scrollbar to Modal for Advanced Settings when Input Limit Exceeded

### PR DESCRIPTION
**Pull request by Cristhianzl**
Description:
This pull request proposes the addition of a scrollbar to the modal window containing advanced settings when the number of input fields exceeds the limit of the screen. This enhancement aims to improve user experience by providing a convenient way to access all settings, even when the available screen space is limited.
The number of inputs in the screen is limited by a variable on Utils that can be easily changed in the future if  It's needed.

### CODE REVIEW

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
When the user is allowed to scroll, the scrollbar should become visible.

File: src/frontend/src/modals/NodeModal/components/ModalField/index.tsx
Comment (anovazzi1) [inline src/frontend/src/modals/NodeModal/components/ModalField/index.tsx:1]: I think we can make this last item pb-3 in a better way than iterating throw all the items every item to check if it's the last

**Review by anovazzi1 — COMMENTED** [review]
could you please tell the reference node with a scrolling issue to test this pr?

**Comment by ogabrielluiz** [comment]
> could you please tell the reference node with a scrolling issue to test this pr?

OpenAIEmbeddings

**Comment by anovazzi1** [comment]
tested locally and looks good, could apply some style on the sidebar, the same as the one in the chat input text-area
image

**Pull request closed without merging**